drivers/lora/sx12xx_common.c

```diff
@@ -169,6 +169,27 @@ static void sx12xx_ev_tx_timed_out(void)
 	modem_release(&dev_data);
 }
 
+static void sx12xx_ev_rx_error(void)
+{
+	struct k_poll_signal *sig = dev_data.operation_done;
+
+	/* Receiving in asynchronous mode */
+	if (dev_data.async_rx_cb) {
+		/* Start receiving again */
+		Radio.Rx(0);
+		/* Don't run the synchronous code */
+		return;
+	}
+
+	/* Finish synchronous receive with error */
+	if (modem_release(&dev_data)) {
+		/* Raise signal if provided */
+		if (sig) {
+			k_poll_signal_raise(sig, -1);
```

Comment:
As this value ends up being returned from `sx12xx_lora_recv`, it would be better to provide a more descriptive error, otherwise users might believe the error is `#define EPERM 1         /**< Not owner */`.

I think either `-ECANCELED` or `-EIO` would be more suitable.

Reply:
Done